Extend/Controller/Admin/PaymentMain.php

```diff
@@ -263,15 +296,19 @@ private function _validateRequestParameters($aParams)
         $sUser = $aParams['oxpayments__wdoxidee_httpuser'];
         $sPass = $aParams['oxpayments__wdoxidee_httppass'];
 
-        if ($sUrl && $sUser && $sPass) {
+        if ($aParams['oxpayments__oxid'] === PayolutionInvoicePaymentMethod::getName(true)) {
```

Comment:
@tomazpu this Payolution check has been existing before (see few lines below), but had to be moved in its own if-statement to make WPP v2 work as expected.